### DISCUSSION TITLE
v1.0.0 @ledgerhq/wallet-api-client & @ledgerhq/wallet-api-client-react not working.

### DISCUSSION THREAD

**Discussion by kallen-ledger** [LedgerHQ/wallet-api#201] [Q&A]
Hey 👋 

I was attempting to set up a react based live app with the `@ledgerhq/wallet-api-client` and `@ledgerhq/wallet-api-client-react` packages.
However, it seems there may be an issue where the `v1.0.0` `@ledgerhq/wallet-api-client` and `@ledgerhq/wallet-api-client-react` NPM packages have not been output correctly. 

The package sizes also appear larger compared to previous versions.

I've tried troubleshooting this in my own fork with [this diff](https://github.com/LedgerHQ/wallet-api/compare/main...kallen-ledger:wallet-api:main?expand=1), which now works when importing a locally built version.

For context, I followed the instructions [here](https://wallet.api.live.ledger.com/react/getting-started) and was using the [vite](https://vitejs.dev/) framework.

Please let me know if I can provide any other information to assist with debugging this issue, I'm happy to help if I can!

**Comment by fenil233** [comment]
Hey @kallen-ledger  I am also facing issue while calling an API it gives error `{code: -32601, message: 'Method not found'}`, any workaround ?

**Reply by ComradeAERGO** [reply]
@fenil233 This issue mentioned by @kallen-ledger has been solved in the following versions of the package. 

You shouldn't experience it in [@ledgerhq/wallet-api-client@1.1.0](https://github.com/LedgerHQ/wallet-api/releases/tag/%40ledgerhq%2Fwallet-api-client%401.1.0) 
 and [@ledgerhq/wallet-api-client-react@1.0.4](https://github.com/LedgerHQ/wallet-api/releases/tag/%40ledgerhq%2Fwallet-api-client-react%401.0.4) 
 
Which versions are you using?

**Reply by fenil233** [reply]
@ComradeAERGO I am using wallet-api-client 1.1.0, and trying both account & wallet APIs at client side but both giving same response `Method not found`

**Reply by ComradeAERGO** [reply]
In that case, your issue is most probably unrelated to @kallen-ledger 's.

Please tell us more about your issue:
- Show us your implementation
- What did you call to raise that error?

**Reply by blackpanther1881** [reply]
hey @ComradeAERGO check this https://github.com/LedgerHQ/wallet-api/discussions/244
